aion/web/channels/question_chronicle.ex

```diff
@@ -5,13 +5,19 @@ defmodule Aion.QuestionChronicle do
     fires, it's easy to check if the question should be changed
     to a new one.
   """
-  @type t :: %{binary => integer}
+  @type t :: %{binary => agent_entry_t}
+  @type agent_entry_t :: {integer, room_state_t}
+  @type room_state_t :: :question | :break
 
-  @question_timeout_seconds 10
-  @question_timeout_micro @question_timeout_seconds * 1_000_000
+  @question_timeout 10
```

Comment:
That feels like it should be part of a config.

Reply:
Definitely, all of the time constants from `room_channel.ex` should also be moved over there. I'll make an issue for that.